solidity/contracts/BondedECDSAKeepFactory.sol

```diff
@@ -68,7 +68,7 @@ contract BondedECDSAKeepFactory is IBondedECDSAKeepFactory, CloneFactory {
     uint256 public constant minimumBond = 1;
 
     // Gas required for a callback from the random beacon. The value specifies
-    // gas required to call `setGroupSelectionSeed` function in the worst-case
+    // gas required to call `__beaconCallback` function in the worst-case
     // scenario with all the checks and maximum allowed uint256 relay entry as
     // a callback parameter.
     uint256 public constant callbackGas = 41830;
```

Comment:
Does the gas value remain the same after changes on the beacon side?

Reply:
It is the callback gas on our side, I don't see a reason why it could change but I'll double-check.

Reply:
The gas estimate was correct but I realized we can get a discount of 10k, see https://github.com/keep-network/keep-ecdsa/pull/339/commits/7b04db60c466ec375cc60296ca987290b4e7f617

You can play with it yourself, just remember about using beacon as a sender so that the modifier in callback lets you go:
```
const BondedECDSAKeepFactory = artifacts.require("BondedECDSAKeepFactory.sol")
const BN = web3.utils.BN;

module.exports = async function () {
  try {
    let factory = await BondedECDSAKeepFactory.at("0x7741C7E704388bFf29f078bF95d04A86C534Bce9")

    let newEntry = new BN(2).pow(new BN(256)).subn(2)

    console.log('estimating...')
    let gas = await factory.__beaconCallback.estimateGas(newEntry)
    console.log("gas estimate: " + gas)

    console.log('calling...')
    let tx = await factory.__beaconCallback(newEntry)
    console.log(tx);
  } catch (error) {
    console.log(error)
  }
}
```